tests/phpunit/tests/option/networkOption.php

```diff
@@ -199,6 +259,8 @@ public function test_get_network_option_sets_notoptions_if_option_not_found() {
 	 * @covers ::update_network_option
```

Comment:
This change will solve the unit test issue.

```suggestion
	 * @group ms-required
         *
	 * @covers ::update_network_option
```